openkaito/tasks.py

```diff
@@ -119,7 +118,7 @@ def random_eth_denver_segments(
     return segments
 
 
```

Comment:
Nit: need to remove "ETH Denver 2024". Maybe just do "in an Ethereum conference"?